src/content/configuration/resolve.md

```diff
@@ -39,7 +40,7 @@ module.exports = {
 
 ### `resolve.alias`
 
-`object`
+`object` `[string]` `boolean: false`
```

Comment:
It doesn't take a `string` as a value I guess. Please check its `key : value` pair

Reply:
I think you meant
`resolve.alias[key] : [string] / false` ? if so, please change the docs accordingly